src/cobalt.rs

```diff
@@ -125,6 +125,10 @@ pub fn build(config: &Config) -> Result<()> {
     // fall back to the default date
     posts.sort_by(|a, b| b.date.unwrap_or(default_date).cmp(&a.date.unwrap_or(default_date)));
 
+    if &config.post_order == "asc" {
```

Comment:
Random notes for the future
- This is a `post_order` in the root  of the yaml file.  Eventually I'll get back to cobalt (from liquid) and will generalize posts so it will instead be something like "collections.posts.order = asc" (in pseudo-config)
- This should switch to being an enum and we'll want to switch to serde *but* by switching to serde, the case of the fields might change due to Rust's style guide.  Not fully sure what to do about it.
  - We can hold off on input validation until that point